app/assets/javascripts/directives/miq_calendar.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * The 'miq-calendar' anngular calendar directive takes care of:
```

Comment:
@Fryguy We will address the prefix after @mzazrivec gets back in 2 weeks.  @AparnaKarve will fix the typo in another PR next week since we found a minor bug in the schedule editor while testing this PR.  I'm going to merge this as is because it does successfully bring in the new datepicker.